### PR TITLE
commit of cart and wishlist pages

### PR DESCRIPTION
**Pull request by mmanjuprasad**
I have created a cart and wishlist  pages please review it 
also Please review the product listing and home pages

### CODE REVIEW

File: cart&wishlist/cart.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.footer button{
+    color: #0346fcbd;
+    font-size: 1.1rem;
+    border:none;
+    background-color: transparent;
+    text-transform: uppercase;
+    font-weight: 700;
+    padding: 0.2rem;
```
Comment (omkarpatke) [inline cart&wishlist/cart.css:18]: if the value is less than 1rem use px.

File: cart&wishlist/cart.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    font-weight: normal;
+    
+  }
+
+
+
+.card-hz{
+    max-width: 350px;
+    border: 1px solid #f7f7f8;
+    border-radius: 0.5rem;
```
Comment (omkarpatke) [inline cart&wishlist/cart.css:44]: here as well

File: cart&wishlist/cart.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    /* flex-basis: 70%; */
+    /* overflow: hidden; */
+
+  }
+  
+  .card-hz .card-header-img {
+    object-fit: cover;
+    flex-basis: 50%;
+    width: 80px;
+    height: 100px;
```
Comment (omkarpatke) [inline cart&wishlist/cart.css:59]: here you want to use **rem** if the value is less than 1rem then use **px** the best practice is rem as Vikram told us.

**Review by omkarpatke — CHANGES_REQUESTED** [review]
do changes in all CSS as I suggest, else other code looks good.

File: cart&wishlist/cart.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  .link{
+    box-sizing:border-box;
+    text-decoration:none;
+    padding: 0.5rem 1rem;
+    
+  }
+  .link-primary{
+    background-color: rgba(0, 88, 189, 0.911);
+    border-radius: 0.5rem;
+    color:white;
```
Comment (Krushna-Kulkarni) [inline cart&wishlist/cart.css:105]: Try to use color variables.

File: cart&wishlist/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                 <span><i class="fas fa-bars fa-lg"></i></span>
+                 
+                 <h1 class="nav-title"> Zeta Kart</h1>
+            </section>
+            <label for="search" class="">
+                <input class="input-search " type="text" placeholder="Search">
+            </label>
+            <section class="nav-container-endsec">
+                <a href="/cart&wishlist/wishlist.html">
+                    <i class="fas fa-heart fa-lg"  style="user-select: auto;"></i>
```
Comment (Krushna-Kulkarni) [inline cart&wishlist/wishlist.html:29]: Avoid inline styling.

File: index.html [hunk 6]
```diff
@@ -23,9 +23,16 @@ <h1 class="nav-title"> Zeta Kart</h1>
                 <input class="input-search " type="text" placeholder="Search">
             </label>
             <section class="nav-container-endsec">
-                <span><i class="fas fa-heart fa-lg"  style="user-select: auto;"></i></span>
-                <span><i class="fas fa-shopping-cart fa-lg"></i></span>
-                <span><i class="fas fa-user-circle fa-lg"></i></span>
+                <a href="/cart&wishlist/wishlist.html">
+                    <i class="fas fa-heart fa-lg"  style="user-select: auto;"></i>
```
Comment (Krushna-Kulkarni) [inline index.html:27]: here as well.

**Review by Krushna-Kulkarni — COMMENTED** [review]
Looks good. suggested some minor changes.

**Pull request merged**